### ISSUE TITLE
NEST Command Index (helpindex.html) remains empty with python2.7

### ISSUE THREAD

**Issue by janhahne** [nest/nest-simulator#1074]
Since the merge of #1062 the NEST Command Index (helpindex.html) remains empty (without any entries) when using python2.7.

There is no error during `make` or `make install` but from executing `generate_help.py` manually I assume that the reason is
```
Traceback (most recent call last):
  File "generate_help.py", line 135, in <module>
    line = html.escape(line)
AttributeError: 'module' object has no attribute 'escape'
```

From looking at https://docs.python.org/3/library/html.html it seems that `escape` has been added with python3.2 and is not available for 2.7.

I also noticated `helpindex.html` is not created at all when the html module is not installed (there is also no error message during make/make install in this case). 

@clinssen any idea how to fix this?

**Comment by clinssen** [comment]
@janhahne: thanks for your feedback! Could you please try to install the "future" package (`pip install future`) to see if that resolves your issue?

**Comment by janhahne** [comment]
@clinssen Thank you for the suggestion. Installing the "future" package indeed resolves the issue for me. 

So the only problem left in my opinion is, that a user without the html and the future package installed has no clue why the helpindex is empty or not generated, since everything fails silently.

**Comment by clinssen** [comment]
It looks like pynest does not have any package dependencies, apart perhaps from numpy which is imported in `hl_api_connections.py`: https://github.com/nest/nest-simulator/blob/18f25d59905da821f0ddc9efc9e9200a84bc76da/pynest/nest/lib/hl_api_connections.py#L31

It might go too far to add infrastructure to check Python version and request/auto-install the html package. I will file a PR that simply uses the "cgi" module instead of "html", which is in both the Python2 and 3 standard library.

**Comment by janhahne** [comment]
Fixed by #1076
